src/API/Helpers/State/SessionStateHandler.php

```diff
@@ -34,14 +34,19 @@ public function __construct(StoreInterface $store)
     }
 
     /**
-     * Generate state value to be used for the state param value during authorization.
+     * Generate state value to be used for the state param value during authorization 
+     * if one does not already exist or we want to force the generation of one.
      *
      * @return string
      */
-    public function issue()
+    public function issue($force = false)
```

Comment:
When would a developer need to use `$force`?

Reply:
if they want to implement a try catch for the getUser method, and want to redo the login if the user is null, that would be a good use case.